lib/run-compilation.js

```diff
@@ -46,7 +46,11 @@ module.exports = function runCompilation (options, compiler, done) {
     }
 
     if (errors.length) {
-      compilation.errors.push(chalk.red(options.formatter(errors)));
+      if (options.emitErrors === false) {
+        compilation.warnings.push(chalk.yellow(options.formatter(errors)));
```

Comment:
If you mean to display them in red, then that was my first try. However, it was rather confusing: usually the red means you have to immediately do something, and that the compilation has failed. Thus, if e.g. another, actual error occurred, one might think the style error would have to be fixed as well.

I therefore kept it yellow, since for all intents and purposes, when you've enabled this flag errors _are_ warnings. I'd be willing to change this if you feel red is more appropriate or if you want to prefix it with something, though. (Or feel free to change it yourself.)